arroyo/processing/strategies/run_task_with_multiprocessing.py

```diff
@@ -605,6 +611,9 @@ def __init__(
         self.__metrics.gauge(
             "arroyo.strategies.run_task_with_multiprocessing.processes", num_processes
         )
+        self.__metrics.gauge(
+            "arroyo.strategies.run_task_with_multiprocessing.pool_size", num_processes
+        )
```

Comment:
These two are redundant. 
Please keep only one. Also you do not need to record the metric here. Only below when you record the processes in use.
If you decide to keep `pool_size` and get rid of `processes` you have to register the metric here https://github.com/getsentry/arroyo/blob/20614a579f4e74a0f4ea2455192894da5ee53f8d/arroyo/utils/metric_defs.py#L47